catalog/app/containers/Bucket/requests/requestsUntyped.js

```diff
@@ -628,8 +628,6 @@ export const summarize = async ({ s3, handle: inputHandle, resolveLogicalKey })
     const manifest = JSON.parse(json)
     const configErrors = isValidManifest(manifest)
     if (configErrors.length) {
-      // eslint-disable-next-line no-console
-      console.error(configErrors[0])
```

Comment:
probably it would still be helpful to see actual errors given by the validator to not guess what's wrong with your config

Reply:
Yes, I should stop sending PR at night